apps/src/aichat/views/model-customization-workspace.module.scss

```diff
@@ -9,17 +9,23 @@
 
 .modelCustomizationWorkspace {
   @extend %panel-section;
-  height: 100%;
   display: flex;
+  flex: 1;
+  overflow: hidden;
   flex-direction: column;
   background: $light_white;
 
   .tabsContainer {
-    background-color: #EAEBEB
+    background-color: #eaebeb;
   }
 
   .tabPanels {
-    padding: 16px;
+    padding: 16px 0;
+    flex: 1;
+    overflow: hidden;
+  }
+
+  .tabPanel {
```

Comment:
instead of adding className - can be simplified here to just styling the nearest level div (see screenshot), which will save us some lines of repeated code
<img width="455" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/22244040/3c08b83e-f8e4-4f73-b780-533ee5158ae9">

Reply:
Ah good call, much simpler - will do!